redcap_is_complete column is required but not available in processing_df

```pytb
[ERROR] AttributeError: 'Series' object has no attribute 'redcap_is_complete'
Traceback (most recent call last):
  File "/var/task/lambda_code.py", line 1450, in lambda_handler
    processing_df = submit_libraries_to_pieriandx(processing_df)
  File "/var/task/lambda_code.py", line 327, in submit_libraries_to_pieriandx
    processing_df["submission_arn"] = processing_df.apply(
  File "/var/lang/lib/python3.9/site-packages/pandas-1.4.3-py3.9-linux-x86_64.egg/pandas/core/frame.py", line 8845, in apply
    return op.apply().__finalize__(self, method="apply")
  File "/var/lang/lib/python3.9/site-packages/pandas-1.4.3-py3.9-linux-x86_64.egg/pandas/core/apply.py", line 733, in apply
    return self.apply_standard()
  File "/var/lang/lib/python3.9/site-packages/pandas-1.4.3-py3.9-linux-x86_64.egg/pandas/core/apply.py", line 857, in apply_standard
    results, res_index = self.apply_series_generator()
  File "/var/lang/lib/python3.9/site-packages/pandas-1.4.3-py3.9-linux-x86_64.egg/pandas/core/apply.py", line 873, in apply_series_generator
    results[i] = self.f(v)
  File "/var/task/lambda_code.py", line 329, in <lambda>
    if x.is_validation_sample or (x.is_research_sample and not x.redcap_is_complete)
  File "/var/lang/lib/python3.9/site-packages/pandas-1.4.3-py3.9-linux-x86_64.egg/pandas/core/generic.py", line 5575, in __getattr__
    return object.__getattribute__(self, name)
```

--- a/deploy/cttso-ica-to-pieriandx-cdk/lambdas/launch_available_payloads_and_update_cttso_lims_sheet/lambda_code.py
+++ b/deploy/cttso-ica-to-pieriandx-cdk/lambdas/launch_available_payloads_and_update_cttso_lims_sheet/lambda_code.py
@@ -171,6 +171,8 @@ def get_libraries_for_processing(merged_df) -> pd.DataFrame:
       * library_id
       * portal_wfr_id
       * is_validation_sample
+      * is_research_sample
+      * redcap_is_complete
     """
 
     # Initialise
@@ -179,7 +181,8 @@ def get_libraries_for_processing(merged_df) -> pd.DataFrame:
         "library_id",
         "portal_wfr_id",
         "is_validation_sample",
-        "is_research_sample"
+        "is_research_sample",
+        "redcap_is_complete"
     ]
 
     # Processing libraries must meet the following criteria
@@ -313,6 +316,8 @@ def submit_libraries_to_pieriandx(processing_df: pd.DataFrame) -> pd.DataFrame:
       * library_id
       * portal_wfr_id
       * is_validation_sample
+      * is_research_sample
+      * redcap_is_complete
     :return:
     """
     # Get number of rows to submit
@@ -344,10 +349,12 @@ def submit_libraries_to_pieriandx(processing_df: pd.DataFrame) -> pd.DataFrame:
         logger.info(f"SubjectID='{row.subject_id}', LibraryID='{row.library_id}', Workflow Run ID='{row.portal_wfr_id}'")
         logger.info(f"Submitted to arn: '{row.submission_arn}'")
         try:
-            submit_library_to_pieriandx(subject_id=row.subject_id,
-                                        library_id=row.library_id,
-                                        workflow_run_id=row.portal_wfr_id,
-                                        lambda_arn=row.submission_arn)
+            submit_library_to_pieriandx(
+                subject_id=row.subject_id,
+                library_id=row.library_id,
+                workflow_run_id=row.portal_wfr_id,
+                lambda_arn=row.submission_arn
+            )
         except ValueError:
             pass
         else:

